### ISSUE TITLE
Add Ability to Run Antora on Playbook .yml Files from VSCODE GUI (Right-Click > Launch Antora)?

### ISSUE THREAD

**Issue by bai-yi-bai** [asciidoctor/asciidoctor-vscode#834]
Hi, I frequently open up the vscode terminal just to type `npx antora antora-playbook.yml`. I'd really like to be able to just right-click a .yml file and click "Launch Antora". 

I use Antora Assembler, with a single repository which has my template and a .yml file for each document. Our output is currently PDF-focused, so each time we have a new document, I copy our template .yml file and make a few modifications. This will lead to a few hundred .yml files.

I apologize for being ignorant of vscode's capabilities. Would this be something I'd be able to write myself?
I found something here:
https://stackoverflow.com/questions/46714353/how-to-customize-context-menu-in-visual-studio-code

**Comment by ggrossetie** [comment]
One way would be to add a `scripts` in a `package.json` file.:

![image](https://github.com/asciidoctor/asciidoctor-vscode/assets/333276/e64b38a4-c994-4acd-8bad-d568f9363947)

**Comment by bai-yi-bai** [comment]
Neat, is there a way to pass a file name through to this script?
For example, I have a lot of playbooks in one folder, each one points to a different repo with a different repository. This is where my AsciiDoctor PDF theme lives. I'm not doing anything fancy with symbolic links.

**Comment by bai-yi-bai** [comment]
hello, disregard my previous comment. I implemented my theme as a git submodule, so now I add that to all my content repos. This allows me to put the antora-playbook.yml and antora-assembler.yml next to each other.

However, I found that the proposed solution doesn't work 100% of the time. I got it working once, but then it complained that I needed to add the NPM extension... but that one is deprecated.

The solution was to create a tasks.json

`
{
    "name": "Render Antora HTML and AsciiDoctorPDF Output",
    "version": "2.0.0",
    "label": "Render Command",
    "type": "shell",
    "command": "npx antora antora-playbook.yml",
}
`

I recycled this from a .bat file I was trying to write, so I also had "presentation" and "group" keys, but I am not sure they are necessary.

Now pressing control+shift+b runs the command.

Great!
